ui-core/src/styles/inputs/slider.css

```diff
@@ -0,0 +1,72 @@
+.range-wrapper {
+  margin: 50px;
```

Comment:
it’s the height of region where you can interact with the slider. 
I can set a way to override it, but by default it’s makes it hard to even click on it if it,s not set.

Reply:
Not sure to understand. Interaction with the input depends on the input height (can be increased with a padding-block if really we think it's hard to click on the slider, we should check with @thibautsailly how he wants this) but not on the margin of the wrapper.

On the other hand, we should maybe change the `height` of the wrapper, because for now it looks like this : 
![Capture d’écran 2024-12-03 à 19 06 38](https://github.com/user-attachments/assets/628f51b0-0825-493b-a146-dc3ac4aa98b9)
The wrapper height doesn't include the input itself, it's a little weird.

Reply:
hm sorry, I think I misread, ok I remove the margin of 50px, it should have been only in the story, not the slider itself.